plutus-tx/src/PlutusTx/AsData.hs

```diff
@@ -61,7 +61,7 @@ becomes
     ]))
     where Ex2 a1 a2 = Example (mkConstr 1 [toBuiltinData a1, toBuiltinData a2])
 
-  {-# COMPLETE Ex1, Ex2 #-}
+  {\-# COMPLETE Ex1, Ex2 #-\}
```

Comment:
This doesn't look right

Reply:
Turns out this is correct escaping. I fixed codeblock rendering as well